packages/backend/src/graphql/project/queries/projectsQueryField.test.ts

```diff
@@ -161,4 +178,30 @@ describe('members', () => {
       ],
     })
   })
+
+  it('should return projects of members when the user is an admin', async () => {
+    const testServer = getTestServer({ userId: '4' })
```

Comment:
hmm, it looks like even if I query userId: 1 here test still succeedes

Reply:
The test still behaved correctly with your changes of userId: 1, who is a member, because the user is querying for his own projects, and the test succeeds. But if you had queried projects from the admin (userId: 4) as a member (userId: 1), it should fail.

Reply:
Oh, I see now, thanks!
On a side note, when I try to query projecrs from userId 4 as userId 1, I get an empty array. Maybe have GraphQl error here?